include/constants/battle_script_commands.h

```diff
@@ -135,47 +135,54 @@ enum MoveEndEffects
     MOVEEND_SYNCHRONIZE_TARGET,
     MOVEEND_ABILITIES,
     MOVEEND_ABILITIES_ATTACKER,
-    MOVEEND_STATUS_IMMUNITY_ABILITIES,
+    MOVEEND_STATUS_IMMUNITY_ABILITIES, // TODO: Do berries come before????
     MOVEEND_SYNCHRONIZE_ATTACKER,
     MOVEEND_ATTACKER_INVISIBLE,
     MOVEEND_ATTACKER_VISIBLE,
     MOVEEND_TARGET_VISIBLE,
     MOVEEND_ITEM_EFFECTS_TARGET,
-    MOVEEND_ITEM_EFFECTS_ALL,
+    MOVEEND_ITEM_EFFECTS_ATTACKER_1,
     MOVEEND_SYMBIOSIS,
-    MOVEEND_KINGSROCK, // These item effects will occur each strike of a multi-hit move
     MOVEEND_SUBSTITUTE,
     MOVEEND_SKY_DROP_CONFUSE,
     MOVEEND_UPDATE_LAST_MOVES,
     MOVEEND_MIRROR_MOVE,
     MOVEEND_DEFROST,
     MOVEEND_NEXT_TARGET, // Everything up until here is handled for each strike of a spread move
+    MOVEEND_HP_THRESHHOLD_ITEMS_TARGET, // Activation only during a multi hit move / ability (parental bond))
     MOVEEND_MULTIHIT_MOVE,
     MOVEEND_MOVE_BLOCK,
-    MOVEEND_ITEM_EFFECTS_ATTACKER,
+    MOVEEND_ITEM_EFFECTS_ATTACKER_2,
     MOVEEND_ABILITY_BLOCK,
-    MOVEEND_SHEER_FORCE, // If move is Sheer Force affected, skip until Opportunist
+    MOVEEND_SHEER_FORCE, // If move is Sheer Force affected, skip to Eject Pack
     MOVEEND_COLOR_CHANGE, // Color Change / Berserk / Anger Shell
-    MOVEEND_RED_CARD, 
+    MOVEEND_KEE_MARANGA_HP_THRESHOLD_ITEM_TARGET,
+    MOVEEND_RED_CARD,
     MOVEEND_EJECT_BUTTON,
-    MOVEEND_LIFEORB_SHELLBELL, 
+    MOVEEND_LIFEORB_SHELLBELL,
```

Comment:
```suggestion
    MOVEEND_LIFE_ORB_SHELL_BELL,
```
Would be more consistent (with referencfes changed accordingly)